### DISCUSSION TITLE
When did /buyclaimblocks get removed?

### DISCUSSION THREAD

**Discussion by justEli** [GriefPrevention/GriefPrevention#2372] [Q&A]
Sorry if I'm late to the party but I recently updated GriefPrevention from a very old version and it seems like /buyclaimblocks has been removed? I can't find anything about it anywhere. Even if I open the plugin's source code and do a global search, nothing matches "buy". Nothing can be found in the config about it anymore as well, no "economy" section, nothing. Where did it go?

**Comment by bigpresh** [comment]
Removed in March, by 2d4b89ecbea63d5f650ac904949aa04526d27722 in PR #2268 for issue #1151

I *think* there's an addon floating about out there that implements them instead.  Otherwise, it'd be easy enough to have someone implement one, or to just implement it via whatever economy-type system you use to run commands on purchase, e.g. `AdjustBonusClaimBlocks`.

**Reply by RoboMWM** [reply]
This is the case for master which is on v17, which has not been published yet. The economy integration still exists in legacy (v16).

**Comment by RoboMWM** [comment]
Where did you download GriefPrevention? Make sure you download from the releases section, not from a ci or similar.

**Reply by justEli** [reply]
Oh right, I tend to fork and make some changes. But, I guess /buyclaimblocks won't be included in the next release then?

**Reply by QarthO** [reply]
Nope, but I do currently have an addon in development that will add lots of functionality to claimblocks, buying, trading, selling, item vouchers, etc

**Reply by RoboMWM** [reply]
The economy integration and most other features will continue to be maintained in the legacy version, which will still be supported until v20 is released.
